cgk_calcium_tools/filehandler.py

```diff
@@ -24,29 +27,42 @@ def ifstr2list(x) -> list:
     return [x]
 
 
+def timer(method):
+    def timed(self, *args, **kwargs):
+        start_time = perf_counter()
+        result = method(self, *args, **kwargs)
+        end_time = perf_counter()
+        elapsed_time = end_time - start_time
+        print(f'{method.__name__} executed in {timedelta(seconds=elapsed_time)}')
+        self.total_time += elapsed_time
+        return result
+    return timed
```

Comment:
sorry, I don't get it , usually the notebooks tell you how much time is used in each cell.